docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -0,0 +1,164 @@
+--------
+
+In this tutorial, you will register a notebook using the Dockstore GitHub App, which automatically syncs your notebook from GitHub to Dockstore. More information about the Dockstore GitHub App can be found :doc:`here </getting-started/github-apps/github-apps-landing-page>`.
+
+Dockstore Notebooks takes advantage of Google Colab's integration with GitHub. The integration allows you to develop your notebook in Google Colab and push your changes to GitHub from Google Colab. With the Dockstore GitHub App installed on your repository, any edits made to your notebook from Google Colab will automatically be synced to Dockstore.
+
+Setting up the Google Colab and GitHub integration
+--------------------------------------------------
+
+To get started, you will need a Google Colab account and a GitHub account.
+
+In your Google Colab account, link your Google Colab account to your GitHub account. Click on the Settings gear icon in the upper right corner, then select GitHub in the dialog box. Click on Authorise with GitHub and authorize Google Colab to access your GitHub account.
+
+.. figure:: /assets/images/docs/google-colab-authorize-with-github.png
+    :width: 60 %
+
+Next, we will set up a notebook on Google Colab so that it's also stored in one of your GitHub repositories. If you already have a notebook in Google Colab that's saved to a GitHub repository, you may proceed to :ref:`create the .dockstore.yml file for your repository<create-dockstore-yml>`.
+
+We will cover two scenarios:
+
+1. :ref:`You don't have a notebook in GitHub<no-notebook-in-github>`. You may or may not have a notebook in Google Colab.
+2. :ref:`You have a notebook in GitHub, but it's not in Google Colab<no-notebook-in-google-colab>`
+
+.. _no-notebook-in-github:
+
+You don't have a notebook in GitHub
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you don't have a notebook in GitHub *nor* Google Colab, navigate to Google Colab and create a new notebook by clicking on File > New notebook. Edit your new notebook if you like.
+
+If you already have a notebook in Google Colab, proceed to the next step.
+
+Click on File > Save a copy in GitHub to save your notebook to a GitHub repository. You may want to create a new repository for your notebook if you don't already have a repository that you want to save the notebook to.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github.png
+    :width: 35 %
+
+Configure where you want to save the notebook to on GitHub then click OK. Google Colab will push a commit to your repository with the notebook.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github-dialog.png
+    :width: 60 %
+
+.. _no-notebook-in-google-colab:
+
+You have a notebook in GitHub, but it's not in Google Colab
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click on File > Open notebook then select the GitHub tab in the dialog box. Select the notebook that you want to import from GitHub. This will open the notebook in Google Colab.
```

Comment:
Alternatively, couldn't you create the .dockstore.yml first, then launch with Colab from Dockstore, and that will guide you through the import? No need to change, just curious, since I've never done it.

Reply:
I think that would work (having trouble testing it right now because of GitHub issues), although starting from Google Colab ensures that you're importing a notebook that is your own before we start registering it. Also from a tutorial POV, it's easier to stay in Google Colab to set up the notebooks vs registering then launching it from Dockstore